src/App.js

```diff
@@ -1,5 +1,18 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import Calculator from "./Calculator.js"; // Calculator 임포트
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      const input = await MissionUtils.Console.readLineAsync("덧셈할 문자열을 입력해 주세요.\n");
+      const calculator = new Calculator();
+      const result = calculator.calculate(input);
```

Comment:
```suggestion
      const result = calculator.getResult(input);
```
`calculator.calculator`가 중복되어서 약간 어색한 부분이 있는데 네이밍을 변경해보는건 어떨까요??